hack/tools/lint-dockerfiles.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -eu
+
+HADOLINT_VERSION=${1:-latest}
+
+FILES=$(find . -path './vendor' -prune -o -name Dockerfile -print)
+
+IFS=$'\n'
```

Comment:
Please comment about the need for this, so future maintenance on the script can determine when/if/how to modify the value